packages/create-svelte/test/check.js

```diff
@@ -21,7 +21,7 @@ const overrides = { ...existing_workspace_overrides };
 	await glob(fileURLToPath(new URL('../../../packages', import.meta.url)) + '/*/package.json')
 ).forEach((pkgPath) => {
 	const name = JSON.parse(fs.readFileSync(pkgPath, 'utf-8')).name;
-	overrides[name] = path.dirname(path.resolve(pkgPath));
+	overrides[name] = `file:${path.dirname(path.resolve(pkgPath))}`
```

Comment:
this should fix the lint failure

```suggestion
	overrides[name] = `file:${path.dirname(path.resolve(pkgPath))}`;
```